cmd/sonarExecuteScan.go

```diff
@@ -374,6 +372,7 @@ func loadCertificates(certificateList []string, client piperhttp.Downloader, run
 			"-keystore", trustStoreFile,
 		}
 		tmpFolder := getTempDir()
+		os.MkdirAll(trustStorePath, 0700)
```

Comment:
`0700` might create issues when cleaning up the file system on Jenkins.

We should use permissions consistently.
Example: https://github.com/SAP/jenkins-library/blob/master/cmd/getConfig.go#L224 prevents this

Reply:
Okay changed it to `0777`